crowdin/messages.json

```diff
@@ -217,7 +217,6 @@
   "339879944": "GBP/USD",
   "341589956": "Air France KLM SA",
   "342146000": "<0>Saving to your computer</0>",
-  "342722863": "\n                Learn more about our commission plans on <0>our Affiliate and IB Programme page</0>.",
```

Comment:
do you know if these \n was due to the new extracting method? I wonder how this can affect translators seeing this as new text. I believe they could be confused with \n characters. Can you adjust to not include this, only the string to traslate?

Reply:
- These keys were generated because of the way the developer introduced the key to `Localize` component. it's not related to the new approach. 
- since we generate the key based on the default value of the string unfortunately I can't change it in any manner and the developer should handle it. I'm thinking about an `eslint` rule for our projects.

Reply:
i understand, and I see that there is also another PR that addresses it. LGTM